File: Loop and pattern C/Question 9.cpp
//Write a program make a summation of given number (E.g., 
#include<stdio.h>
int main(){
	int i,n[5];
	
	for(i=0;i<5;i++){
		printf("Enter the value of element %d: ",i);
		scanf("%d",&n[i]);
		
	}
	printf("The summation is %d",n[0]+n[1]+n[2]+n[3]+n[4]);
}

File: Python/Assignment/Module 2/Question9.py
'''
Write a Python program to sum of three given integers.
However, if two values are equal sum will be zero.
'''
a=int(input("Enter Integer 1:"))
b=int(input("Enter Integer 2:"))
c=int(input("Enter Integer 3:"))

if(a==b or b==c or c==a):
    print("0")
else:
    print("The sum is :",a+b+c)    